src/front/spv.rs

```diff
@@ -10,11 +10,16 @@ TODO: would be nice to find ways that avoid looking up as much
 
 !*/
 
+use std::convert::TryFrom;
+
+use crate::front::spv_dag::*;
 use crate::{
     arena::{Arena, Handle},
     FastHashMap, FastHashSet,
 };
 
+use daggy::petgraph::*;
```

Comment:
Would be great to avoid doing this as well. I know this is a quick prototype, just leaving a note for us to not forget about this :)